### PR TITLE
README.md - deprecation/re-direction

### PR DESCRIPTION
**Pull request by JanneKiiskila**
The code is not maintained in this repo anymore, but via the Mbed Cloud Client offering instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## Deprecation note!
+
+**Please note this repository is deprecated - this repository is no longer actively maintained**. 
+
+However, this library is part of Arm's [Mbed Cloud](https://cloud.mbed.com/docs/current) -offering and as such the code is available via [Mbed Cloud Client](https://github.com/ARMmbed/mbed-cloud-client-example) under
```
Comment (teetak01) [inline README.md]: instead of mbed-cloud-client-example this should link to mbed-cloud-client repository.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## Deprecation note!
+
+**Please note this repository is deprecated - this repository is no longer actively maintained**.
```
Comment (iriark01) [inline README.md]: How about:

**Please note: this repository is deprecated and is no longer actively maintained**

File: README.md (hunk 1 above)
Comment (iriark01) [reply README.md]: Why is there - before "offering"?
Also, you can shorten it to "offering, and the code is available in [give just one link, directly to the correct folder]

**Review by teetak01 — APPROVED** [review]
I added some fixes.

**Pull request merged**